cdc/redo/manager.go

```diff
@@ -317,35 +290,33 @@ func (m *ManagerImpl) FlushLog(
 	tableID model.TableID,
 	resolvedTs uint64,
 ) error {
```

Comment:
The function name `FlushLog()` is no longer related to log flushing. It's better to use a different name.

Reply:
Rename it to `UpdateResolvedTs`, just like [sinkV2](https://github.com/pingcap/tiflow/blob/master/cdc/sinkv2/tablesink/table_sink.go#L28) did.